phpseclib/System/SSH/Agent.php

```diff
@@ -128,7 +128,7 @@ function __construct()
                 break;
             default:
                 user_error('SSH_AUTH_SOCK not found');
-                return false;
+                return;
```

Comment:
Oh, right. I just noticed a breaking change in the diff, and didn;t check the context. :)